src/components/Calculator.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable  no-unused-vars */
```

Comment:
- [ ] <b>REQUIRED:</b> As mentioned in the previous review, you are over using`eslint-disable`. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them.

Reply:
I am not over using it!!
I was haveing an unnecessary error: the class I imported was marked as unused var; and since there is nothing I could do about it, I have to use /* eslint-disable  no-unused-vars */ just for a singular error.